packages/schematics/angular/config/index.ts

```diff
@@ -103,3 +106,77 @@ function addKarmaConfig(options: ConfigOptions): Rule {
     );
   });
 }
+
+interface ContextFileInfo {
+  rulesName: string;
+  directory: string;
+  frontmatter?: string;
+}
+
+function addAiContextFile(options: ConfigOptions): Rule {
+  const files: ContextFileInfo[] = [];
+
+  const geminiFile: ContextFileInfo = { rulesName: 'GEMINI.md', directory: '.gemini' };
```

Comment:
Can these constants be moved to a module level constant object literal?
That would also remove the need for the switch and also allow for easier expansion/adjustment of tools.